numba_cuda/numba/cuda/tests/cudapy/test_ipc.py

```diff
@@ -21,80 +22,69 @@
 import unittest
 
 
-def core_ipc_handle_test(the_work, result_queue):
-    try:
-        arr = the_work()
-    # Catch anything going wrong in the worker function
-    except:  # noqa: E722
-        # FAILED. propagate the exception as a string
-        succ = False
-        out = traceback.format_exc()
-    else:
-        # OK. send the ndarray back
-        succ = True
-        out = arr
-    result_queue.put((succ, out))
-
-
-def base_ipc_handle_test(handle, size, result_queue):
-    def the_work():
-        dtype = np.dtype(np.intp)
-        with cuda.open_ipc_array(
-            handle, shape=size // dtype.itemsize, dtype=dtype
-        ) as darr:
-            # copy the data to host
-            return darr.copy_to_host()
-
-    core_ipc_handle_test(the_work, result_queue)
-
-
-def serialize_ipc_handle_test(handle, result_queue):
-    def the_work():
-        dtype = np.dtype(np.intp)
-        darr = handle.open_array(
-            cuda.current_context(),
-            shape=handle.size // dtype.itemsize,
-            dtype=dtype,
-        )
+def base_ipc_handle_test(handle, size, parent_pid):
+    pid = os.getpid()
+    assert pid != parent_pid
```

Comment:
Passing the `parent_pid` here isn't strictly necessary and I can remove it if desired, I was just verifying my understanding of what was happening with `ProcessPoolExecutor`.